src/layouts/AppHeader.js

```diff
@@ -1,65 +1,119 @@
 import React from 'react';
-import { Layout, Menu } from 'antd';
+import { Icon, Layout, Menu } from 'antd';
 import { FormattedMessage } from 'react-intl';
 import { UserContext } from '../UserContext';
 import { Link } from "react-router-dom";
 import logo from '../assets/logo.png';
 import './AppHeader.less';
 
 const { Header } = Layout;
+const { SubMenu } = Menu;
 
 class AppHeader extends React.Component {
   render() {
     return (
       <UserContext.Consumer>
-        { ({switchLang}) => {
+        { ({switchLang, isLoggedIn}) => {
             return (
               <Header className="app-header">
-                  <img src={logo} className="app-header-logo" alt=""/>
+                <img src={logo} className="app-header-logo" alt=""/>
+                { isLoggedIn ?
                   <Menu
                     theme="dark"
                     mode="horizontal"
-                    defaultSelectedKeys={["explore"]}
```

Comment:
Remove the selected status really is the right way?
To have a currect select behaviour:
Method 1: AppHeader takes an argument "current page". i.e. the Home page passes the "home".

Method 2: Move the AppHeader outside the routering part (need to handle the dashboard part carefully) and https://blog.csdn.net/qq_40190624/article/details/88907350.